pokemon-checker/src/components/PokemonDisplay/DisplayFunctionalComponents/StatDisplay.tsx

```diff
@@ -0,0 +1,75 @@
+import "../PokemonDisplayStyle.css";
```

Comment:
This is a good start, we definitely need something that will render the stat numbers.

In the future I can see this component getting subviews to handle graphs via D3 or another library. We can talk about that offline.